src/components/organisms/Carousel/Carousel/ Carousel.test.tsx

```diff
@@ -0,0 +1,93 @@
+import { fireEvent, render, screen } from '@testing-library/react';
```

Comment:
Is it me or is that an extra character in the filename?

Reply:
![lens](https://media1.tenor.com/images/2c3daa52c432316895513616a7fdd7a2/tenor.gif)